docs/book/src/security/infrastructure-provider-security-guidance.md

```diff
@@ -0,0 +1,12 @@
+# Infrastructure Provider Security Guidance
```

Comment:
I would add a brief intro, eg.

```suggestion
# Infrastructure Provider Security Guidance

There are few critical areas that any infrastructure provider implementer should take care of in order to ensure secure operations:
- Management of cloud credentials assigned to the infrastructure provider, including setting corresponding quotas, rate limiting, etc.
- Ensure secure access to VM for troubleshooting
- Ensuring control of all the manual operations performed on the cloud infrastructure targeted by the provider
- Housekeeping of the cloud infrastructure targeted by the provider

Following list is a community maintained list of high level recommendations; everyones help is welcome and necessary to continuously improve such list.

Each provider implementer is responsible to translate those recommendations in the context of each cloud provider.


```

Reply:
It is addressed in commit a512d28. Please re-review. Thanks